learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+# Week 3 Learning Outcomes
+
+## Testing
+
+- [ ] **We can test the functionality of our app**
+- [ ] We can write tests to mimic the behaviour of a user performing different actions
+- [ ] We can check that passing a given input into our tests returns the expected output
+
+### Modular functions
+
+- [ ] **We can write testable, modular functions**
+- [ ] We can write functions that add DOM nodes
+- [ ] We can write functions that remove DOM nodes
+- [ ] We can write functions that modify DOM nodes
+
+### Event listeners
+
+- [ ] **We can apply click event listeners to buttons**
+- [ ] We can pass in named functions as the second argument
+
+### Scope
+
+- [ ] **We can apply the concept of ‘scoping’ to functions and variables**
```

Comment:
"apply the concept" might be a little vague/confusing. Can we think of a concrete application for scoping? Like "We can use function scope to..."

Reply:
How about "We can use function and block scope to store values inside temporary variables"?

I agree, the more concrete examples we can scaffold these LOs around, the better